docs/en/observability/synthetics-feature-roles.asciidoc

```diff
@@ -0,0 +1,124 @@
+[[synthetics-feature-roles]]
+= Grant users access to secured resources
+
+You can use role-based access control to grant users access to secured
+resources. The roles that you set up depend on your organization's security
+requirements and the minimum privileges required to use specific features.
+
+Typically you need the create the following separate roles:
+
+* <<synthetics-setup-privileges,Setup role>> for enabling Monitor Management.
+* <<synthetics-write-privileges,Writer role>>  for creating, modifying, and deleting monitors.
+* <<synthetics-read-privileges,Reader role>> for {kib} users who need to view and
+create visualizations that access Synthetics data.
+
+{es-security-features} provides {ref}/built-in-roles.html[built-in roles] that grant a
+subset of the privileges needed by Synthetics users.
+When possible, assign users the built-in roles to minimize the affect of future changes on your security strategy.
+If no built-in role is available, you can assign users the privileges needed to accomplish a specific task.
+
+In general, these are types of privileges you'll work with:
+
+* **{es} cluster privileges**: Manage the actions a user can perform against your cluster.
+* **{es} index privileges**: Control access to the data in specific indices your cluster.
+// * **{kib} space privileges**: Grant users write or read access to features and apps within {kib}.
+
+[discrete]
+[[synthetics-setup-privileges]]
+= Setup role
+
+Administrators who set up Synthetics typically need to enable Monitor Management.
```

Comment:
I wonder if there's a way to give them more context about what setup means. Not sure if we have another doc with a screenshot of the setup flow.

Reply:
Good idea! We mention this setup step in the "get started" guides, but a screenshot would probably be helpful, too.